nwutils/nw-builder
# build(compressing): unzip via `yauzl-promise` on MacOS

Fixes: #931

## Description

- Removed workaround which requires `unzip` library on MacOS.
- Use `yauzl-promise` to unzip MacOS build on MacOS platform.

## Testing steps

If using in a existing project:

HTTPS:
```shell
npm i -D git+https://github.com/nwutils/nw-builder/tree/dev-931
```

GIT:
```shell
npm i -D git+ssh://git@github.com/nwutils/nw-builder/tree/dev-931
```

Standalone:

1. Clone repo
2. Checkout `dev-931` branch
3. `npm i` to install depencies
4. `npm run demo` to download binaries
5. Check if symlinks are preserved
